docs/docs/en/guide/installation/rainbond.md

```diff
@@ -0,0 +1,72 @@
+# Use Rainbond Deployment
+
+This article describes the one-click deployment of highly available DolphinScheduler clusters through the [Rainbond](https://www.rainbond.com/) cloud native application management platform,This method is suitable for users who don't know much about complex technologies such as Kubernetes,Lowered the threshold for deploying DolphinScheduler in Kubernetes.
+
+## Prerequisites
+
+* Available Rainbond cloud native application management platform，Please refer to the documentation [Rainbond Quick install](https://www.rainbond.com/docs/quick-start/quick-install)
+
+## DolphinScheduler Cluster One-click Deployment 
+
+* Docking and accessing the built-in open source app store,Search the keyword `dolp` to find App DolphinScheduler.
+
+![](https://static.goodrain.com/wechat/dolphinscheduler/1.png)
```

Comment:
can you add this img to our `img` directory? we need to hold those images instead of using extracting link

Reply:
No problem